docs/man/upsd.conf.txt

```diff
@@ -32,6 +32,19 @@ When instant commands and variables setting status tracking is enabled, status
 execution information are kept during this amount of time, and then cleaned up.
 This defaults to 3600 (1 hour).
 
+"ALLOW_NO_DEVICE 'Boolean'"::
```

Comment:
I would mention the environment variable as well, and the fact that it takes precedence over the conf file (realizing that this is mentioned in the sample file)

Reply:
Thanks, done.